### PR TITLE
Fix/create draft payrun inputs

### PR DESCRIPTION
**Pull request by mugnivenko**

### CODE REVIEW

File: components/Pages/Finance/CreateDraftPayRun/CreatePayRunInfo.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
+import { Controller } from 'react-hook-form';
+import React from 'react';
+import { Container, FormGroup, RadioGroup } from '../../../HackneyDS';
+import DatePick from '../../../DatePick';
+
+const CreatePayRunInfo = ({
```
Comment (akeamwoodsND) [inline components/Pages/Finance/CreateDraftPayRun/CreatePayRunInfo.js:6]: Break this up into two components
Screenshot 2021-11-11 at 14 06 33
and
Screenshot 2021-11-11 at 14 06 58
It is better to have two simpler components than one complicated one with lots of props. I'd expect this component to look something like

```
<Modal>
<RegularCycles/>
<AdHocAndReleased/>
<Container>
<Button>Cancel</Button>
<Button>Create draft pay run</Button>
</Container>
</Modal>
```

**Review by akeamwoodsND — CHANGES_REQUESTED** [review]
See comment

**Comment by mugnivenko** [comment]
Not full tested yet

File: components/Pages/Finance/CreateDraftPayRun/CreatePayRunInfo.js (hunk 1 above)
Comment (akeamwoodsND) [reply components/Pages/Finance/CreateDraftPayRun/CreatePayRunInfo.js:5]: Having this component makes the code way more complicated, strip the logic into the other two files.

**Review by akeamwoodsND — CHANGES_REQUESTED** [review]

File: components/Pages/Finance/CreateDraftPayRun/CreatePayRunInfo.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
+import { Controller } from 'react-hook-form';
```
Comment (akeamwoodsND) [inline components/Pages/Finance/CreateDraftPayRun/CreatePayRunInfo.js:1]: Delete this component

**Review by akeamwoodsND — CHANGES_REQUESTED** [review]

**Review by akeamwoodsND — CHANGES_REQUESTED** [review]
Also please revert the changes on the tabs, just use the default tabs without style changes. I know the figma design has it how you have made it, but that is **not** following the HDS which we should be doing
Screenshot 2021-11-15 at 10 42 32

**Review by akeamwoodsND — CHANGES_REQUESTED** [review]
Screenshot 2021-11-15 at 10 43 56
Screenshot 2021-11-15 at 10 44 12
Let's put better validation messages.

**Review by akeamwoodsND — CHANGES_REQUESTED** [review]
edit: ignore

**Pull request merged**